docs/guides/manage-translations.rst

```diff
@@ -197,21 +197,23 @@ Once you have done changes in your documentation, you may want to make these add
 
 #. Create the ``.pot`` files:
 
-   .. code-block:: console
+   .. prompt:: bash $
 
-      $ sphinx-build -b gettext . _build/gettext
+      sphinx-build -b gettext . _build/gettext
 
 
-.. For the manual workflow, we need to run this command
+   For the manual workflow, we need to run this command
 
-      $ sphinx-intl update -p _build/gettext -l es_AR -l pt_BR
+   .. prompt:: bash $
```

Comment:
No need to `.. prompt:` in this case since it's a comment.

Reply:
Thanks.
I wasn't aware of the syntax for comments.
I changed it.